pkg/qemu-ga/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.5
```

Comment:
OK, it's a bit roundabout but I reckon I can follow the pattern in the `dhcpcd` package. Might be a while though